### PR TITLE
[PORT] Adds the oranges' juicery barsign

### PR DESCRIPTION
**Pull request by ktlwjec0**
# Document the changes in your pull request
https://github.com/tgstation/tgstation/pull/79580
Adds a new bar sign.

# Why is this good for the game?
From the original PR: We need options for the non-alcoholics around us, and refreshing orange juice is an old favourite.

# Testing
image

# Spriting
orangejuice

:cl: Thunder12345
rscadd: New Orange Juicery bar sign for the OJ connoisseurs.
/:cl:

### CODE REVIEW

**Comment by ThatGuyPhil** [comment]
looks like the perfect barsign to use for serving screwdrivers exclusively

File: code/game/machinery/barsigns.dm [hunk 1]
```diff
@@ -440,6 +440,12 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/barsign, 32)
 	desc = "First stop out of hell, last stop before heaven."
 	neon_color = COLOR_RED
 
+/datum/barsign/orangejuice
+	name = "Oranges' Juicery"
```
Comment (Moltijoe) [inline code/game/machinery/barsigns.dm]: Pretty sure we've tried to avoid referencing orange, and this could be seen as one

Comment (ktlwjec0) [reply code/game/machinery/barsigns.dm]: sure, changed

**Comment by cuackles** [comment]
sorry but this is a tg players/coder reference and not even one of our regulars so im not comfortable leting this pr pass trougth

**Pull request closed without merging**